com.unity.multiplayer.mlapi/Runtime/Core/NetworkManager.cs

```diff
@@ -872,11 +865,13 @@ public void Shutdown()
                 NetworkTickSystem.Tick -= OnNetworkManagerTick;
                 NetworkTickSystem = null;
             }
-
+            // NSS TODO: Remove this once MTT-860 is addressed or before PR
```

Comment:
MTT-860 Might not get addressed before this PR and I might just remove the NSS TODO portion but leave the reference to MTT-860. The scenario is if a user reloads a scene (i.e. game over and restart a new game while still in the same session) that has the NetworkManager within it and the NetworkManager has Don't Destroy enabled then another NetworkManager instance is created causing a conflict between the existing NetworkManager in the DoNotDestroyOnLoad scene and the one being instantiated when the scene loads.
This check prevents the newly instantiated NetworkManager that is being destroyed from stopping the profiler if the NetworkManager instance shutting down is not actively listening when it was shutdown.